sg_default/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "vpc_id" {}
+variable "sg_name" {}
+variable "sg_description" {
+  default = "Security Group managed by Terraform"
+}
+variable "inbound_rules" {
+  type = "map"
+}
+variable "outbound_rules" {
+  type = "map"
+  default = {
```

Comment:
not sure if there is a good reason to keep such default value. What if a user needs to close outbound access for the security group?